test/unit/app/browser/reducers/downloadsReducerTest.js

```diff
@@ -257,4 +277,19 @@ describe('downloadsReducer', function () {
       })
     })
   })
+
+  describe('APP_SET_STATE', function () {
+    it('DOWNLOAD_DEFAULT_PATH is empty', function () {
```

Comment:
Can you update this to match the `it` syntax? Like `it('sets DOWNLOAD_DEFAULT_PATH when empty', function () {`

Reply:
addressed in b2f581ca0ec3f461bb27afacd1e3539963c84ad6